litestar/app.py

```diff
@@ -24,6 +24,7 @@
 from litestar.config.app import AppConfig, ExperimentalFeatures
 from litestar.config.response_cache import ResponseCacheConfig
 from litestar.connection import Request, WebSocket
+from litestar.contrib.opentelemetry import OpenTelemetryInstrumentationMiddleware
```

Comment:
This will break for all users that don't have the `opentelemetry` package installed

Reply:
I haven't thought of that.
and yes 
```
try:
    import opentelemetry  # noqa: F401
except ImportError as e:
    raise MissingDependencyException("opentelemetry") from e
```
will cause this to break